pages/user-interfaces.html

```diff
@@ -0,0 +1,163 @@
+extension</a> that is actively maintained by Maxime Dénès as part
+of <a href="https://github.com/coq-community/manifesto">coq-community</a>.
+VsCoq was initially authored by C.J. Bell, but was left unmaintained
+for several years before being forked in coq-community.
+</li>
+
+<li>
+<strong>Emacs</strong> users can use the major Coq
+mode <a href="https://proofgeneral.github.io/">Proof General</a>, that
+can be extended by the minor Coq
+mode <a href="https://github.com/cpitclaudel/company-coq">Company-Coq</a>.
+Proof General was one of the first user interface for proof
+assistants, and is one of the most used user interface for Coq today.
+Proof General and Company-Coq give access to many productivity
+shortcuts, including auto-completion and documentation.  On the other
+hand, async proof processing is not supported.
+</li>
+
+<li>
+<strong>Vim</strong> users can use the actively
+maintained <a href="https://github.com/whonore/Coqtail">Coqtail</a>
+extension.  This extension is partly based upon a previous, and now
+unmaintained, Vim extension
+called <a href="https://github.com/the-lambda-church/coquille">Coquille</a>.
+</li>
+
+<li>
+<strong>NeoVim</strong> users can use Julien
+Lepiller's <a href="https://framagit.org/tyreunom/coquille">Coquille
+fork.</a>
+</li>
+
+</ul>
+
+</p>
+
+<p>
+An experimental alternative to the editors mentioned above is the
+computational notebook interface provided by Jupyter.
+A <a href="https://github.com/EugeneLoy/coq_jupyter/">Jupyter kernel
+for Coq</a> is available if you want to try this interface, for
+instance for pedagogical or readability purposes.  One of the
+advantages of using a Jupyter notebook is that you can save and
+display a selection of intermediate proof steps, which your reader
+will see without the need to reexecute the document.
+</p>
+
+</div>
+
+<div class="frameworklinks">
+<ul>
+<li><a class="extlink" href="https://github.com/coq-community/vscoq">VsCoq (VsCode)</a></li>
+<li><a class="extlink" href="https://proofgeneral.github.io/">Proof General (Emacs)</a></li>
+<li><a class="extlink" href="https://github.com/whonore/Coqtail">Coqtail (Vim)</a></li>
+<li><a class="extlink" href="https://github.com/EugeneLoy/coq_jupyter">Jupyter kernel</a></li>
+</ul>
+</div>
+</div>
+
+<div class="framework">
+<div class="frameworklabel">Standalone interfaces</div>
+<div class="frameworkcontent">
+
+<p>
+If you don't want to use any of the previously mentioned editors, you
+can
+use <a href="https://coq.inria.fr/refman/practical-tools/coqide.html">CoqIDE</a>,
+which is developed and distributed alongside Coq.  Like most of the
+more modern editor extensions, CoqIDE relies
+on <a href="https://github.com/coq/coq/blob/master/dev/doc/xml-protocol.md">Coq's
+XML protocol for IDEs</a>, and implements all of the features provided
+by this protocol.  On the other hand, it does not support automatic
+indentation contrarily to most editor extensions.
```

Comment:
I would prefer to remove this sentence and to replace it by something of the form:

CoqIDE comes with standard basic Emacs bindings and with some specific Coq-related bindings but, as an editor, it is not as complete as a general purpose editor could be (for instance it does not have automatic indentation).

To be discussed of course, if opinions diverge.

Reply:
I like the formulation you propose.